src/schema/v2/verify_address.ts

```diff
@@ -0,0 +1,125 @@
+import {
+  GraphQLInt,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLUnionType,
+  GraphQLString,
+  GraphQLEnumType,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ResolverContext } from "types/graphql"
+import {
+  GravityMutationErrorType,
+  formatGravityError,
+} from "lib/gravityErrorHandler"
+
+const ErrorType = new GraphQLObjectType<any, ResolverContext>({
+  name: "VerifyAddressMutationFailure",
+  isTypeOf: (data) => {
+    return data._type === "GravityMutationError"
+  },
+  fields: () => ({
+    mutationError: {
+      type: GravityMutationErrorType,
+      resolve: (err) => (typeof err.message === "object" ? err.message : err),
+    },
+    statusCode: {
+      type: GraphQLInt,
+      resolve: (result) => result.statusCode,
+    },
+    message: {
+      type: new GraphQLNonNull(GraphQLString),
+      resolve: () => "Address could not be verified.",
+    },
+  }),
+})
+
+const SuccessType = new GraphQLObjectType<any, ResolverContext>({
+  name: "VerifyAddressMutationSuccess",
+  isTypeOf: (data) => data.success,
+  fields: () => ({
+    verificationStatus: {
+      type: new GraphQLEnumType({
+        name: "VerificationStatuses",
+        values: {
+          VERIFIED_NO_CHANGE: { value: "VERIFIED_NO_CHANGE" },
+          VERIFIED_WITH_CHANGES: { value: "VERIFIED_WITH_CHANGES" },
+          VERIFICATION_UNAVAILABLE: { value: "VERIFICATION_UNAVAILABLE" },
+          NOT_FOUND: { value: "NOT_FOUND" },
+        },
+      }),
+      resolve: (result) => result.verificationStatus,
+    },
+    inputAddress: {
+      type: new GraphQLObjectType<any, ResolverContext>({
+        name: "AddressInput",
+        fields: addressInputFields,
+      }),
+      resolve: (result) => result.inputAddress,
+    },
+    suggestedAddresses: {
+      type: new GraphQLNonNull(GraphQLString),
+      resolve: (result) => result.suggestedAddresses,
+    },
+  }),
```

Comment:
I don't think this is a string. It might be somewhat complex though. Would it help if I looked over what we return now to define the types better?

Reply:
actually that would be very helpful!